tests/test_generators/input/kitchen_sink.yaml

```diff
@@ -60,6 +60,10 @@ classes:
       aliases:
         multivalued: true
 
+  God:
+    slots:
+      - deprecated
```

Comment:
I'm not sure what you are trying to do here. I know it's a fun test example but sometimes this can lead to confusion

You have a class God, which means that you are saying there are many God instances - say: Thor, Ganesh, Dave, ...

The class God has slot "deprecated" (which is confusingly the same name as a slot from the metamodel), so you can declare individual instances as deprecated...

Let's keep this simple. Let's only introduce one change to kitchensink, under classes:

```yaml
   Patient:
     is_a: Person
     deprecated: deprecated because we want to treat this as a role rather than a subclass
```

Reply:
Hey Chris. Ah, I see. I think this is just a misunderstanding of LinkML. I thought adding `deprecated` as a slot was the way to do it, but I guess it's defined in the metamodel. If deprecated is not a slot, I guess it is a "keyword"? As for God, it's just a toy example; Anything else that makes sense is good.

I'll make this change.

I think this is why I was having an issue where the generator wasn't detecting it as deprecated; because I'd put it as a slot, which is not the correct way to do this.

Reply:
formally deprecated is a slot:

[linkml:deprecated](https://w3id.org/linkml/deprecated)

it's just that it's a slot in the metamodel not in your domain model. The meta stuff gets a bit confusing and the docs may be inconsistent in how these kinds of things are described!